fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/ContextualCommandBarTokens.kt

```diff
@@ -0,0 +1,213 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme.aliasTokens
+import com.microsoft.fluentui.theme.FluentTheme.themeMode
+import com.microsoft.fluentui.theme.ThemeMode
+import com.microsoft.fluentui.theme.token.*
+import kotlinx.parcelize.Parcelize
+
+
+@Parcelize
+open class ContextualCommandBarTokens : ControlToken, Parcelable {
+
+    companion object {
+        const val Type: String = "ContextualCommandBar"
+    }
+
+    @Composable
+    open fun keyboardDismissBackgroundColor(): Color {
```

Comment:
Here, generic name should be used as the actionButton could perform anyother action as well.